### PR TITLE
right-align numbers in tables

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** right-align numbers in tables, round-up values, update authen/end user values
- **Is this PR related to a Monolith release?** Yes 2020.10.0

### Resolves:

* [OKTA-333858](https://oktainc.atlassian.net/browse/OKTA-333858)

### CODE REVIEW

**Review by bridgetwent-okta — COMMENTED** [review]
Some of these values have too many significant figures, which makes it sort of hard to look at in my opinion. Can we round the values to 2 sig-figs to keep things cleaner?

cc @peteryaworsky-okta @karthicksolaimalai-okta

File: packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md [hunk 1]
```diff
@@ -35,9 +35,9 @@ For concurrent rate limits, traffic is measured in three different areas. Counts
 * For Office365 traffic, the limit is 75 concurrent transactions per org.
 * For all other traffic, including API requests, the limit is described in the table below.
 
-| Developer (free) | Developer (paid) | One App | Enterprise | Workforce Identity |
-| ---------------- | ---------------- | ------- | ---------- | ------------------ |
-| 15               | 35               | 35      | 75         | 75                 |
+| Developer (free)  | Developer (paid)  | One App  | Enterprise  | Workforce Identity  |
```
Comment (haozhao-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md:38]: nit: curious why an extra space added here?

**Review by haozhao-okta — COMMENTED** [review]
LGTM, can you also add a figure of the new page?

File: packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md (hunk 1 above)
Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md:38]: that was just me being OCD and making it readable in the markdown. Doesn't affect the live display of the table.

**Comment by susanharper-okta** [comment]
@haozhao-okta Here are images of the updated rate limit tables:
Rate Limits Overview page
Screen Shot 2020-09-25 at 4 47 15 PM
Authentication/End-User
Screen Shot 2020-09-25 at 4 46 32 PM
Management
Screen Shot 2020-09-25 at 4 47 50 PM

**Review by bridgetwent-okta — APPROVED** [review]
Thank you for correcting the rounding/formatting! Are we going to check in these changes only once @haozhao-okta 's changes are merged as part of the monthly release?

**Review by haozhao-okta — APPROVED** [review]

**Comment by susanharper-okta** [comment]
> Thank you for correcting the rounding/formatting! Are we going to check in these changes only once @haozhao-okta 's changes are merged as part of the monthly release?

@bridgetwent-okta I think that is the plan, now. To do it all with the monthly release.

File: packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md [hunk 2]
```diff
@@ -21,20 +21,20 @@ This page provides the API rate limits for authentication and end-user activitie
 | Action and Okta API Endpoint                                                                                           | Developer (free) | Developer (paid) | One App | Enterprise | Workforce Identity    |
 | ---------------------------------------------------------------------------------------------------------------------- | ----------------: | ----------------: | -------: | ----------: | ---------------------: |
 | **Cumulative rate limit**                                                                                              | **1,600**        | **3,800**        | **4,200**| **10,650**| **13,300**            |
```
Comment (karthicksolaimalai-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md]: Please update the "Cumulative rate limit" section for One App and Dev Paid to match the increase we are doing at the endpoint level.  Pls double check but I think Dev Paid and One App should be at 8,400. @simranbrar-okta, after we decide what to do with /wsfed endpoints for One App, we can talk about combining One App and Enterprise and calling it "Customer Identity".  We should also get Yuliya's feedback on that.

Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md]: @simranbrar-okta - could you please provide the Cumulative rate limit changes? Thank you.

Comment (karthicksolaimalai-okta) [reply packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md]: Correct.  The main hold up is with `/wsfed` that needs analysis before we can decide if we can include it in OneApp. @simranbrar-okta - FYI

Comment (simranbrar-okta) [reply packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md]: @haozhao-okta can you pls provide the cumulative limits based on your PR?

Comment (haozhao-okta) [reply packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md]: Confirmed the cumulative limits for one-app and dev paid are 8400 @simranbrar-okta  - FYI

Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md]: for ALL of them, or just for Auth/end user? What are the values for management and other endpoint categories

**Pull request merged**